src/layout/parser.rs

```diff
@@ -103,10 +117,13 @@ impl GradientType for GradientKind {
         GradientKind::Transparent
     }
     fn parse(kind: &[u8]) -> Result<Self> {
+        // FIXME: Implement delta color support properly:
+        // https://github.com/LiveSplit/livesplit-core/issues/380
+
         Ok(match kind {
-            b"Plain" => GradientKind::Plain,
-            b"Vertical" => GradientKind::Vertical,
-            b"Horizontal" => GradientKind::Horizontal,
+            b"Plain" | b"PlainWithDeltaColor" => GradientKind::Plain,
+            b"Vertical" | b"VerticalWithDeltaColor" => GradientKind::Vertical,
+            b"Horizontal" | b"HorizontalWithDeltaColor" => GradientKind::Horizontal,
```

Comment:
Turns out that I completely forgot that this was a thing. We completely failed parsing those layouts. For now it kind of just ignores it. There's a proper issue tracking support for them.